Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERB = @
+ifeq ($(VERBOSE),1)
+	VERB =
+endif
+
+PYTHON_DIR = third_party/python
```

Comment:
I put a `.gitignore` file inside `third_party/python`, but the directories themselves need to stay, because I need to have an `__init__.py` file in each of the directories to be able to import from them.

Specifically, I need:

* `third_party/__init__.py`
* `third_party/python/__init__.py`

even though they're empty. Both of these files are included in this PR.